UpdateTypes.md

```diff
@@ -28,4 +28,6 @@ Terraria Servers use this to tell Dimensions to switch the clients Dimension
 | Type           | 2    | Int16    |
 | Server IP      | ?    | String   |
 | Server Port    | ?    | UInt16   |
-Terraria Servers use this to tell Dimension to switch the client to a specific ip/port that is not in the Dimensions config.
\ No newline at end of file
+| Hidden         | ?    | Boolean  |
```

Comment:
It doesn't look like you added this in the code for parsing the packet, and I doubt it would be useful since it's not related to the dimensions list. So I would say remove this line since this is just for the packet structure.

Reply:
I created a definition at Server Config Structure and it is enforced to transfer while switching Dimensions, this is why i added this but we can also remove it if its unnecessary.

Reply:
That structure is for the packet a *terraria server* sends to dimensions to switch a client to a specific ip/port. It's not related to the config.

Reply:
Alright, removed in commit 512e5731070e66b83c8ce589646210aa01778403